scripts/chat_thread_summarizer.py

```
@@ -0,0 +1,505 @@
+            except:
+                recent_commits = ["git_log_not_available"]
+
+            # Get MemOS containers specifically
+            try:
+                docker_ps = subprocess.run(
+                    ["docker", "ps", "--filter", "name=memos", "--format", "table {{.Names}}\\t{{.Status}}\\t{{.Ports}}"],
+                    capture_output=True,
+                    text=True
+                )
+                memos_containers = docker_ps.stdout.strip().splitlines() if docker_ps.returncode == 0 else []
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)